docs/development/pr-review.asciidoc

```diff
@@ -0,0 +1,129 @@
+[[pr-review]]
+== Pull request review guidelines
+
+Every change made to Kibana must be held to a high standard, and while the responsibility for quality in a pull request ultimately lies with the author, Kibana team members have the responsibility as reviewers to verify during their review process.
+
+Frankly, it's impossible to build a concrete list of requirements that encompass all of the possible situations we'll encounter when reviewing pull requests, so instead this document tries to lay out a common set of the few obvious requirements while also outlining a general philosophy that we should have when approaching any PR review.
+
+While the review process is always done by Elastic staff members, these guidelines apply to all pull requests regardless of whether they are authored by community members or Elastic staff.
+
+
+[float]
+=== Target audience
+
+The target audience for this document are pull request reviewers. For Kibana maintainers, the PR review is the only part of the contributing process in which we have complete control. The author of any given pull request may not be up to speed on the latest expectations we have for pull requests, and they may have never read our guidelines at all. It's our responsibility as reviewers to guide folks through this process, but it's hard to do that consistently without a common set of documented principles.
+
+Pull request authors can benefit from reading this document as well because it'll help establish a common set of expectations between authors and reviewers early.
+
+
+[float]
+=== Reject fast
+
+This document only seeks to describe the high level goals of the PR review process. It is not expected nor intended for a PR review to take the shape of this document, where the topmost guidelines are always applied first and the bottommost guidelines are always applied last. This entire document should be considered as a whole rather than as a series of consecutive todos that must be completed in order.
```

Comment:
seems like this paragraph belongs at the top

Reply:
I can't tell if you're making a joke here about this being the "reject fast" section :P

This section is basically the first section that actually focuses on the PR review process, so it seems like the right place to me. The prior content all focused on this document itself - what's it about and how should it be read.

Reply:
It would be so meta if we get into a deadlock review situation now...

Reply:
@sqren Ha! I know you're joking, but it's worth emphasizing for anyone following along:

Disagreements are fine, that's what the PR process is for. So long as we focus on progressing towards a consensus in these cases, then all is working as intended.

Reply:
Haha, sorry I was reviewing very quickly to get it in before a meeting.  It's fine if you leave it here, just felt like a general statement rather than something specific to the rejecting fast rule.  I should have written my comment like:

`nit: seems like this paragraph belongs at the top` 😆

Reply:
I misunderstood your feedback and thought it applied to the whole "reject fast" section, but it's clear to me now that you are referring to this specific paragraph. I think you're right, this could be further up.

Reply:
I moved this up to the top section